src/io.cpp

```diff
@@ -87,4 +93,107 @@ bool IO::ok()
     return ros::ok();
 }
 
+bool IO::sendPath(std::vector<std::vector<double>> path, std::array<double, 3> color, double width, int id)
+{
+    MapConfig mapData;
+    comm_->getMapConfig(mapData);
+    visualization_msgs::Marker pathMarker;
+    pathMarker.header.frame_id = "map";
+    pathMarker.header.stamp = ros::Time();
+    pathMarker.ns = "path";
+    pathMarker.id = id;
+    pathMarker.type = visualization_msgs::Marker::LINE_STRIP;
+    pathMarker.action = visualization_msgs::Marker::ADD;
+    pathMarker.color.a = 1.0;
+    pathMarker.color.r = color[0];
+    pathMarker.color.g = color[1];
+    pathMarker.color.b = color[2];
+    pathMarker.pose.orientation.w = 1.0;
+    pathMarker.scale.x = width;
+    for (std::vector<std::vector<double>>::iterator it = path.begin(); it != path.end(); ++it)
+    {
+        geometry_msgs::Point point;
+        if ((*it).size() < 2)
```

Comment:
```suggestion
        if (it->size() != 2)
```
Also break if they added more then 2 values. That is also not intended behaviour and probably indicates that something went wrong.